api/glossary_usefulconcepts.md

```diff
@@ -22,23 +24,40 @@ Examples include JavaScript, Python
 
 **2. Object-orientated programming**
 - This centres around the creation of objects that are characterised by attributes and behaviour. 
-- **Classes** are used in OOP in the creation of objects. High-level versus low-level languages
-- This refers to the level of abstraction. Each level is a step away from the base machine 
+- **Classes** are used in OOP in the creation of objects. 
+- When comparing high-level to low-level languages, this refers to the level of abstraction. Each level is a step away 
```

Comment:
Think you still need the "High-level versus low-level languages" - just in this bullet point?